geomstats/_backend/autograd/autograd.py

```diff
@@ -0,0 +1,14 @@
+"""Wrapper around autograd functions to be consistent with backends."""
```

Comment:
Very nice! It was impossible to do this before, this will make it easier to wrap custom grad!